create-deploy-pull-request/tests/github.ts

```diff
@@ -0,0 +1,6 @@
+import * as github from '@actions/github'
+import { setupServer } from 'msw/node'
+
+export const server = setupServer()
+
+export const getOctokit = () => github.getOctokit('GITHUB_TOKEN', { request: { fetch } })
```

Comment:
It needs to set the global `fetch` function by the below reason.

- msw intercepts the global `fetch` function.
- actions/github uses another fetch implementation by default. https://github.com/actions/toolkit/blob/bb2278e5cfbb40afc20890c415e9ffa836631cd5/packages/github/src/utils.ts#L17